docs/core/tools/dotnet-new.md

```diff
@@ -71,26 +71,26 @@ The command calls the [template engine](https://github.com/dotnet/templating) to
 | Blazor Server App                            | [blazorserver](#blazorserver)   | [C#]         | Web/Blazor                            | 3.0        |
 | ASP.NET Core Empty                           | [web](#web)                     | [C#], F#     | Web/Empty                             | 1.0        |
 | ASP.NET Core Web App (Model-View-Controller) | [mvc](#web-options)             | [C#], F#     | Web/MVC                               | 1.0        |
-| ASP.NET Core Web App                         | [webapp, razor](#web-options)   | [C#]         | Web/MVC/Razor Pages                   | 2.2, 2.0   |
+| ASP.NET Core Web App                         | [webapp, dotnet new ](#web-options)   | [C#]         | Web/MVC/Razor Pages                   | 2.2, 2.0   |
```

Comment:
why put "dotnet new" in the "short name" column?

Reply:
That was a mistake—didn't mean to do that. Sorry.